app/assets/javascripts/pageflow/built_in_widget_types.js

```diff
@@ -8,4 +8,10 @@ pageflow.widgetTypes.register('default_mobile_navigation', {
   enhance: function(element) {
     element.navigationMobile();
   }
+});
+
+pageflow.widgetTypes.register('default_mute_toogle', {
+  enhance: function(element) {
+    element.muteToggle();
+  }
 });
```

Comment:
The new widget is just a jQuery UI widget, not a Pageflow widget type that can be activated or deactivated by the user. This code is not needed.